applications/asset_tracker_v2/src/cloud/cloud_codec/azure_iot_hub_codec.c

```diff
@@ -0,0 +1,380 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <cloud_codec.h>
+#include <stdbool.h>
+#include <string.h>
+#include <zephyr.h>
+#include <zephyr/types.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "cJSON.h"
```

Comment:
This should be `#include <cJSON.h>` as it's a global Zephyr library

Reply:
Will fix